prime-router/src/main/kotlin/fhirengine/translation/hl7/FhirTransformer.kt

```diff
@@ -203,7 +203,7 @@ class FhirTransformer(
                         (childResource as Extension).url = matchResult.groupValues[1]
                     }
                 }
-                else -> childResource = childResource.addChild(childName.replace("""[\[0-9\]]""".toRegex(), ""))
+                else -> childResource = childResource.addChild(childName.replace("""\[[0-9]+\]""".toRegex(), ""))
```

Comment:
This wasn't related to the change, but in looking for potential sources of error, I noticed this regex was slightly off.